### PR TITLE
wraping glfwSetFramebufferSizeCallback

### LINKED ISSUE #207: glfwSetFramebufferSizeCallback mssing in the wrapper
I'd say this isn't necessarily a bug, nor is it a feature request.
I was just wondering about the reason why some of the glfw functions aren't wrapped in Silk.net.glfw and if there was a way to load them anyways?
I was specifically looking for `glfwSetFramebufferSizeCallback` which I presume would be something like `GLFW.SetFramebufferSizeCallback` in the wrapper. I see you got some of the callback setters in there like windowResize / Refresh, so why not all of them?

### PR DESCRIPTION
**Pull request by SaculRennorb**
# Summary of the PR
wraping glfwSetFramebufferSizeCallback + a test project to see that it's working

# Related issues, Discord discussions, or proposals
Closes #207 

# Further Comments
The first commit is the addition, the second one is just the test project. If you don't want this it can easily be reverted.

### CODE REVIEW

**Comment by SaculRennorb** [comment]
jea.. no, i just noticed that the test project is at the completely wrong location, let me know if you want it in here at all and i will fix that.

**Comment by Perksey** [comment]
We keep test projects like that in the lab: `src/Lab`

Fine with keeping it in, just make sure it's moved to that location :)

Will review now.

**Comment by SaculRennorb** [comment]
sry, took a while to unmangle the sln, vs loves to change things in there..
I knew where to put the project from the start, but vs puts projects in the solution root by default, and i didn't notice

File: Silk.NET.sln [hunk 1]
```diff
@@ -289,7 +289,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Extensions", "Extensions",
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Silk.NET.Input.Extensions", "src\Input\Extensions\Silk.NET.Input.Extensions\Silk.NET.Input.Extensions.csproj", "{8DCB2EB4-D0DE-4EF9-ACB3-332055D594CB}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Silk.NET.Windowing.Extensions.Veldrid", "src\Windowing\Extensions\Silk.NET.Windowing.Extensions.Veldrid\Silk.NET.Windowing.Extensions.Veldrid.csproj", "{F5D5B4B4-58B7-48D9-881B-DD6A4001793E}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FramebufferResized", "src\Lab\FramebufferResized\FramebufferResized.csproj", "{7AF3C6CA-64FE-43F3-BD7B-C57C83AFAB79}"
```
Comment (Perksey) [inline Silk.NET.sln:292]: You've accidentally removed the Silk.NET.Windowing.Extensions.Veldrid project from the solution.

```suggestion
Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Silk.NET.Windowing.Extensions.Veldrid", "src\Windowing\Extensions\Silk.NET.Windowing.Extensions.Veldrid\Silk.NET.Windowing.Extensions.Veldrid.csproj", "{F5D5B4B4-58B7-48D9-881B-DD6A4001793E}"
EndProject
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FramebufferResized", "src\Lab\FramebufferResized\FramebufferResized.csproj", "{7AF3C6CA-64FE-43F3-BD7B-C57C83AFAB79}"
```

**Comment by Perksey** [comment]
Oh also the build seems to be failing due to the solution. I recommend just resetting the solution file and using `dotnet sln add` to add the new project back.

**Comment by SaculRennorb** [comment]
already on it, vs didn't like the project move

**Pull request merged**